### DISCUSSION TITLE
Statistics: unable to remove statistics for a non-recorded sensor

### DISCUSSION THREAD

**Discussion by ildar170975** [home-assistant/frontend#16516] [Configuration] [answered]
There is a sensor which has a stored statistics.
Recently this sensor was excluded from Recorder.
Since then this sensor is proposed to be fixed:

![изображение](https://github.com/home-assistant/frontend/assets/71872483/5e0d5642-35a6-49b9-951f-a4970475d029)

But this "fix" is supposed to be "do not exclude from Recorder":
![изображение](https://github.com/home-assistant/frontend/assets/71872483/49e17b88-367f-480d-9d65-3f789af15e41)

I think that there must be a possibility to remove statistics for a sensor intentionally excluded from Recorder.

**Comment by ildar170975** [comment]
2024.6, still an issue.
Exclude some sensor with LTS from recorder:
```
recorder:
  exclude:
    entities:
      - sensor.day_vs_night_ratio
```
Then see this line:
![изображение](https://github.com/home-assistant/frontend/assets/71872483/87fa8dda-ef62-49d3-8e9f-35e949c2efac)

and this useless info:
![изображение](https://github.com/home-assistant/frontend/assets/71872483/c48cda94-b922-411c-90ac-a3ab6abd2797)

**Comment by karwosts** [comment, marked as answer]
The statistics are now deletable after #21119.

**Comment by ildar170975** [comment]
@karwosts 
2024.8
Still see similar window for an entity excluded from Recorder:
![image](https://github.com/user-attachments/assets/38be3868-e9e7-428e-b148-029d62d574c6)

**Reply by ildar170975** [reply]
[You wrote here:](https://github.com/home-assistant/frontend/pull/21119):
>This just replaces one issue with another (entity no longer recorded -> entity not recorded), but at least it cleans up the DB.

OK, but what the point to have these useless issues?
If an entity is excluded intentionally - why to remind about "hmmm, this entity may be LTS-able, but you excluded"?

**Reply by karwosts** [reply]
I couldn't say why someone decided to raise that as an issue. 

Perhaps sometime we'll get the data table updated with the new design, and it will have easy filtering that you can hide issue types you are not interested in.

**Reply by ildar170975** [reply]
> hide issue types you are not interested in.

Surely this would be a useful addition.
But initially - considering this as an "issue" is rather questionable for me
